css/style.css

```diff
@@ -1 +1,1091 @@
-body {}
\ No newline at end of file
+:root {
+  --btn-primary-color: rgba(33, 231, 134, 1);
+  --secondary-color: rgba(20, 27, 34, 1);
+  --article-bg-color: rgba(255, 255, 255, 0.05);
+  --main-background-color: rgb(13, 13, 17);
+  --text-color: rgba(194, 195, 197, 1);
+  --inactive-color: rgba(136, 139, 142, 1);
+  --text-font: "Poppins", sans-serif;
+}
+
+/* Uneversal css */
+
+* {
+  padding: 0;
+  margin: 0;
+  background-color: rgba(4, 11, 17, 1);
+  font-family: "Bakbak One", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 18px;
+  line-height: 28px;
+  color: rgba(255, 255, 255, 1);
+}
+
+a {
+  text-decoration: none;
+  color: rgba(255, 255, 255, 1);
+}
+
+ul {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  padding: 0 255px;
+}
+
+button,
+.btn-link,
+.btn-border {
+  padding: 12px 24px;
+  font-size: 16px;
+  line-height: 24px;
+  gap: 8px;
+  cursor: pointer;
+}
+
+span {
```

Comment:
Better not to style tags like <span> or other elements directly:

Applying styles directly to tags like span can lead to unintended consequences, especially if those tags are used in various contexts throughout the site. It's better to use classes or IDs for more specific targeting, which helps maintain consistent and predictable styling.

Example:

```
.highlighted-text {
    color: red;
}
```

This way, you can control which specific span elements are affected by your styles.

Reply:
#fixed